src/main/webapp/dev/js/common/instructorEnroll.js

```diff
@@ -0,0 +1,160 @@
+/**
```

Comment:
This new file should have been committed in the same commit where the old version is deleted.